### PR TITLE
Add duration field

### PR DESCRIPTION
**Pull request by oliverguenther**
https://community.openproject.org/work_packages/22553/activity
https://community.openproject.org/work_packages/22560/activity

### CODE REVIEW

**Comment by TeatroIO** [comment]
I've prepared a stage to preview changes. <a href='http://oliverguenther-feature-inline-edit-duration.opf-openproject-2af22bf12c7dfdad4ddf.ttrcloud.com/' target='_blank'>Open stage</a> or <a href='http://oliverguenther-feature-inline-edit-duration.opf-openproject-2af22bf12c7dfdad4ddf.ttrcloud.com//teatro.log' target='_blank'>view logs</a>.

**Comment by oliverguenther** [comment]
Supersedes https://github.com/opf/openproject/pull/4217

File: frontend/app/components/wp-edit/wp-edit-field/wp-edit-duration-field.directive.html [hunk 1]
```diff
@@ -0,0 +1,5 @@
+<input type="number"
+       wp-edit-field-requirements="vm.field.schema"
+       ng-model="vm.workPackage[vm.fieldName]"
+       duration-value
+       ng-blur="vm.deactivate()">
```
Comment (ulferts) [inline frontend/app/components/wp-edit/wp-edit-field/wp-edit-duration-field.directive.html]: You should add the `focus` directive to let the field be focused after triggering inline edit.

Comment (oliverguenther) [inline frontend/app/components/wp-edit/wp-edit-field/wp-edit-duration-field.directive.html]: Ah, thanks. done.

**Pull request merged**